twilight-model/src/gateway/event/mod.rs

```diff
@@ -6,6 +6,8 @@ pub mod shard;
 mod dispatch;
 mod kind;
 
+use crate::id::{marker::GuildMarker, Id};
+
```

Comment:
Should be grouped with the other imports

Reply:
That's the only import from the crates root.

Reply:
Yes but Twilight groups the following way:
1. pub mod
2. mod
3. pub use
4. use

So this import should be below `pub use`